app.py

```diff
@@ -45,19 +45,25 @@ class App:
     version: str = "v3"
     demo: bool = True
     model_name: str = "decenter-model-linear-reg-sample_v3"
-
+    model_name_changed: bool = False
+    
+    def set_model_name(self, model_name:str):
+        app.model_name=model_name
+        app.model_name_changed=True
     def validate_model_name(self):
```

Comment:
add a blank line before defining this method

Reply:
done